tests/all/test_main.py

```diff
@@ -98,5 +93,40 @@ def test_argparse_exec_workflow(self):
         self.assertEquals(parsed.func.__name__, "run_cli")
         self.assertTrue(parsed.verbose)
 
+    def test_argparse_exec_ensemble_experiment_current_case(self):
+        parser = ArgumentParser(prog="test_main")
+        parsed = ert_parser(parser, ['ensemble_experiment', "--current-case", 'test_case',
+                                     'test-data/local/poly_example/poly.ert'])
+        self.assertEquals(parsed.mode, "ensemble_experiment")
+        self.assertEquals(
+            parsed.config, "test-data/local/poly_example/poly.ert")
+        self.assertEquals(parsed.current_case, 'test_case')
+        self.assertEquals(parsed.func.__name__, "run_cli")
+        self.assertFalse(parsed.verbose)
+    
+    def test_argparse_exec_ensemble_smoother_current_case(self):
+        parser = ArgumentParser(prog="test_main")
+        parsed = ert_parser(parser, ['ensemble_smoother', "--current-case", 'test_case',
+                                     "--target-case", 'test_case_smoother',
```

Comment:
I would like a test case where the current and target case are the same as well.

Reply:
Yes, we should have an integration test for the CLI. The parser itself is not capable of doing this test, as it needs to load the current storage before it checks current_case vs target_case.

Reply:
Right. Merge this as is and create a new issue, or resolve it in this pr?